theme/templates/base.html

```diff
@@ -128,6 +128,9 @@
             {% block main %}{% endblock %}
             {% block after_main %}{% endblock %}
             {% include "pages/site-wide-modals.html" %}
+
+            {# ======= Uppy for large file upload ======= #}
+            {% include "uppy.html" %}
```

Comment:
added uppy into the base template so that notifications can be given on return to ANY page in HS